content/docs/user-guide/scripting/lua/_index.md

```diff
@@ -10,18 +10,16 @@ You can use Lua in **Open 3D Engine (O3DE)** to facilitate quick iteration of yo
 
 O3DE uses Lua version {{< versions/lua >}}.
 
-## Building Lua Editor and GridHub
+## Lua Editor and GridHub
 
-If you built O3DE from source, you must explicitly include **Lua Editor** (Lua IDE) and GridHub as build targets when you build your project. GridHub is O3DE's central connection hub for debugging, and is required to debug Lua scripts from the Lua IDE. To include these targets, add `LuaIDE` and `GridHub` to your build target list: `--target <project>.GameLauncher Editor LuaIDE GridHub`. 
+The Lua development environment in O3DE includes the **Lua Editor**. The debugger that is included with the Lua Editor uses a connection hub called **GridHub**. These two applications are built when the **O3DE Editor** is built.
```

Comment:
It used to be the case that building the Editor target did *not* build LuaIDE and GridHub, has that changed?

Reply:
Yes! This changed in a recent PR. (Well, two PR's, actually.)

1. https://github.com/o3de/o3de/pull/4514
2. https://github.com/o3de/o3de/pull/5229